src/test/scala-2.12/pbdirect/ProtocComparisonSpec.scala

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
:art:

Reply:
This was added by that damned license headers plugin that I can't figure out how to disable 🙃